Mastodon/Scene/Profile/Header/View/ProfileHeaderView.swift

```diff
@@ -417,7 +428,11 @@ extension ProfileHeaderView {
         // nameMetaText.textView.setContentHuggingPriority(, for: <#T##NSLayoutConstraint.Axis#>)
         
         nameContainerStackView.addArrangedSubview(displayNameStackView)
-        nameContainerStackView.addArrangedSubview(usernameLabel)
+        nameContainerStackView.addArrangedSubview(usernameButton)
+        
+        NSLayoutConstraint.activate([
+            usernameButton.heightAnchor.constraint(equalToConstant: 20)
```

Comment:
Out of curiousity: Have you checked what happens, when users use Dynamic Type? Is the text still visible although the height-constraint is limited to `20`?

Reply:
Very fair point! Should use `greaterThanOrEqualToConstant` probably. But anyways there is currently no change in behavior, as `adjustsFontSizeToFitWidth = true` leads to the username always being shrinked to fit the label width, even in the current release.